app/routes/register.js

```diff
@@ -1,8 +1,15 @@
 import RSVP from 'rsvp';
 import Route from '@ember/routing/route';
+import ENV from 'irene/config/environment';
 
 const RegisterRoute = Route.extend({
   title: 'Registration',
+  beforeModel(){
+    if(!ENV.isRegistrationEnabled) {
+      window.location.href = 'https://try.appknox.com/schedule-appknox-demo';
```

Comment:
This needs to be taken from the environment variable and not hardcoded here

Reply:
Oh okay :+1: got it